style.css

```diff
@@ -903,7 +909,7 @@
 
   .form-section {
     background-image: url(./vectors/contact\ form\ background\ shapes.svg);
-    background-size: 90%;
+    background-size: 1199px;
     background-repeat: no-repeat;
     background-position: center;
     padding-inline: 350px;
```

Comment:
[OPTIONAL] You have done a great job with your portfolio so far 👍 . However, in your mobile version, there seems to be an issue with your input fields and your textarea. They seem to have fixed width values and are not displaying correctly:

![inputs](https://user-images.githubusercontent.com/111214189/229267131-37921855-559c-4aeb-b9a5-650073f0c538.png)

You can simply fix this issue by using a non-fixed width value (for example, use % instead). This issue was missed by the previous reviewer but I highly recommend you implement it. This will make your portfolio look more professional and improve the user experience with your website.

Reply:
![image](https://user-images.githubusercontent.com/111823606/229288581-0abca8fc-9ceb-4110-a67e-703a1d78a258.png)

Got it, thanks!!!